src/models/ComponentStyle.js

```diff
@@ -49,6 +49,8 @@ export default (
   flatten: Flattener,
   stringifyRules: Stringifier
 ) => {
+  const generateName = (str: string) => nameGenerator(hashStr(str))
```

Comment:
This is somewhat confusing with the static version further in the file.

Reply:
I’ll rename it

Reply:
done